src/index.html

```diff
@@ -1,22 +1,276 @@
-<!doctype html>
-<html lang="en">
+<!-- #region head -->
+<!DOCTYPE html>
+<html lang="en" class="page">
   <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Miami</title>
-    <link
-      rel="stylesheet"
-      href="./styles/main.scss"
-    />
-    <script
-      src="scripts/main.js"
-      defer
-    ></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>THE MET</title>
+    <link rel="icon" href="images/favicon.ico.png">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+
+  <body class="main">
+    <header class="header" id="home">
+      <div class="header__bar">
+        <a href="#">
+          <img src="images/favicon.ico.png" alt="logo">
+        </a>
+        <div class="header__icons">
+          <a href="tel:+1 212-535-7710" class="icon icon--phone">
+            <div class="icon icon--phone--hover">
+              <p>The Met Fifth Avenue</p>
+              <p>+1 212-535--7710</p>
+              <p>The Met Cloisters</p>
+              <p>+1 212-923-3700</p>
+            </div>
+          </a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+      <h1 class="header__title">Welcome to the Met</h1>
+    </header>
+
+    <aside class="menu" id="menu">
+      <div class="menu__bar">
+        <a href="#">
+          <img src="images/favicon.ico.png" alt="Logo">
+        </a>
+        <div class="header__icons">
```

Comment:
I believe this element got lost of it's parent kkkk.
Jokes aside, double check this BEM ;)